src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(-n + 0),
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
The `:nth-child(-n + 0)` selector in `.stars--0` will not select any elements because `-n + 0` evaluates to zero, which does not match any child. Consider adjusting this selector to ensure it selects the intended elements.